Makefile-ponyc

```diff
@@ -51,6 +51,16 @@ tune ?= generic
 cpu ?= $(arch)
 fpu ?=
 bits ?= $(shell getconf LONG_BIT)
+staticbuild ?= false
+
+# Set static building if requested
+ifeq (true,$(staticbuild))
```

Comment:
we have a standard format for putting these together elsewhere, it does the filter check first and errors out and then sets based on the value. i think this should be adjusted to do that. there's no logical change, just an approach change.

Reply:
Can you point me to an example of the standard format? I'm too braindead to figure it out at the moment.

Reply:
```make
ifdef staticbuild
  ifeq (,$(filter $(staticbuild), true false))
  	$(error staticbuild must be true or false)
  endif
endif

ifeq ($(staticbuild),true)
  use += llvm_link_static 
endif
```

is how it appears elsewhere in Makefiles that I've been copying as a pattern when I add new items

Reply:
ooooh... i remember that pattern... thanks